data/css/modules/contentGroup/_contentGroup.scss

```diff
@@ -96,7 +96,7 @@
     }
 
     &__paginate {
-        background-color: $brand-color;
+        background-color: $primary-dark-color;
```

Comment:
Ah, I think you squashed this one into the wrong commit - it should be in "Add pagination to content group". This is currently the only thing preventing me from merging the pagination commits to master, since the build breaks for lack of `$brand-color`